ckan/lib/helpers.py

```diff
@@ -2121,125 +2262,35 @@ def get_translated(data_dict, field):
-    'resource_link',
-    'tag_link',
-    'group_link',
-    'dump_json',
-    'auto_log_message',
-    'snippet',
-    'convert_to_dict',
-    'activity_div',
-    'lang_native_name',
-    'get_facet_items_dict',
-    'unselected_facet_items',
-    'include_resource',
-    'urls_for_resource',
-    'build_nav_main',
-    'build_nav_icon',
-    'build_nav',
-    'build_extra_admin_nav',
-    'debug_inspect',
-    'dict_list_reduce',
-    'full_current_url',
-    'popular',
-    'debug_full_info_as_list',
-    'get_facet_title',
-    'get_param_int',
-    'get_translated',
-    'sorted_extras',
-    'follow_button',
-    'follow_count',
-    'remove_url_param',
-    'add_url_param',
-    'groups_available',
-    'organizations_available',
-    'user_in_org_or_group',
-    'dashboard_activity_stream',
-    'recently_changed_packages_activity_stream',
-    'escape_js',
-    'get_pkg_dict_extra',
-    'get_request_param',
-    'render_markdown',
-    'format_resource_items',
-    'resource_preview',
-    'rendered_resource_view',
-    'resource_view_get_fields',
-    'resource_view_is_filterable',
-    'resource_view_is_iframed',
-    'resource_view_icon',
-    'resource_view_display_preview',
-    'resource_view_full_page',
-    'remove_linebreaks',
-    'SI_number_span',
-    'localised_number',
-    'localised_SI_number',
-    'localised_nice_date',
-    'localised_filesize',
-    'list_dict_filter',
-    'new_activities',
-    'time_ago_from_timestamp',
-    'get_organization',
-    'has_more_facets',
-    'roles_translated',
-    # imported into ckan.lib.helpers
-    'literal',
-    'link_to',
-    'get_available_locales',
-    'get_locales_dict',
-    'truncate',
-    'file',
-    'mail_to',
-    'radio',
-    'submit',
-    'asbool',
-    'uploads_enabled',
-    'get_featured_organizations',
-    'get_featured_groups',
-    'get_site_statistics',
-    'get_allowed_view_types',
-    'urlencode',
-    'check_config_permission',
-    'view_resource_url',
-    'license_options',
-]
+builtin_helper(flash, name='flash')
+builtin_helper(localised_number)
+builtin_helper(localised_SI_number)
+builtin_helper(localised_nice_date)
+builtin_helper(localised_filesize)
+# Useful additionsfrom the i18n library.
+builtin_helper(i18n.get_available_locales)
+builtin_helper(i18n.get_locales_dict)
+# Useful additions from the webhelpers library.
+builtin_helper(tags.literal)
+builtin_helper(tags.link_to)
+builtin_helper(tags.file)
+builtin_helper(tags.submit)
+builtin_helper(tools.mail_to)
+builtin_helper(whtext.truncate)
+# Useful additions from the paste library.
+builtin_helper(converters.asbool)
+# Useful additions from the stdlib.
+builtin_helper(urlencode)
+
+
+def load_plugin_helpers():
+    """
+    (Re)loads the list of helpers provided by plugins.
+    """
+    global helper_functions
+
+    helper_functions.clear()
+    helper_functions.update(_builtin_functions)
+
+    for plugin in p.PluginImplementations(p.ITemplateHelpers):
```

Comment:
Are you sure? It seems more intuitive that the latest one overrides the earlier one, as this is how I'd assume it would work. This is how the current, existing _Helper behavior works.